quicktests/overlaying/tests/functional/nearestEntity.js

```diff
@@ -0,0 +1,220 @@
+            barPlot.entityNearest(p).datum.y + ", " +
+            barPlot.entityNearest(p).datum.type);
+        }
+    });
+    pointer.attachTo(barPlot);
+
+    new Plottable.Interactions.PanZoom(xScale, null).attachTo(barPlot);
+
+    return new Plottable.Components.Table([
+        [yAxis, barPlot],
+        [null,  xAxis],
+        [null, title],
+    ]);
+}
+
+function createSegmentPlot() {
+    const plot = new Plottable.Plots.Segment();
+    const xScale = new Plottable.Scales.Linear();
+    const yScale = new Plottable.Scales.Linear();
+    var xAxis = new Plottable.Axes.Numeric(xScale, "bottom");
+    var yAxis = new Plottable.Axes.Numeric(yScale, "left");
+
+    const data = [
+        { x: 1, y: 1, x2: 4, y2: 4 },
+        { x: 2, y: 2, x2: 3, y2: 5 },
+        { x: 3, y: 3, x2: 5, y2: 2 },
+    ];
+    plot.addDataset(new Plottable.Dataset(data));
+    plot.x((d) => d.x, xScale);
+    plot.y((d) => d.y, yScale);
+    plot.x2((d) => d.x2);
+    plot.y2((d) => d.y2);
+
+    const defaultTitleText = "Closest entity";
+    const title = new Plottable.Components.TitleLabel(defaultTitleText);
+
+    const pointer = new Plottable.Interactions.Pointer();
+    pointer.onPointerMove(function(p) {
+        if (plot.entityNearest(p)) {
+            title.text(
+                plot.entityNearest(p).datum.x + ", " +
+                plot.entityNearest(p).datum.y + ", " +
+                plot.entityNearest(p).datum.x2 + ", " +
+                plot.entityNearest(p).datum.y2);
+        }
+    });
+    pointer.attachTo(plot);
+
+    new Plottable.Interactions.PanZoom(xScale, null).attachTo(plot);
+
+    return new Plottable.Components.Table([
+        [yAxis, plot],
+        [null,  xAxis],
+        [null, title],
+    ]);
+}
+
+function createNumericalScatter(data, logarithmic = false) {
+    var plot = new Plottable.Plots.Scatter()
+    let xScale;
+    if (logarithmic) {
+        xScale = new Plottable.Scales.ModifiedLog();
+    } else {
+        xScale = new Plottable.Scales.Linear();
+    }
+    var yScale = new Plottable.Scales.Linear();
+
+    plot.addDataset(new Plottable.Dataset(data))
+        .x(function (d) { return d.x; }, xScale)
+        .y(function(d) { return d.y; }, yScale);
+
+    var xAxis = new Plottable.Axes.Numeric(xScale, "bottom");
+    var yAxis = new Plottable.Axes.Numeric(yScale, "left");
+
+    var defaultTitleText = "Closest entity";
+    var title = new Plottable.Components.TitleLabel(defaultTitleText);
+
+    var pointer = new Plottable.Interactions.Pointer();
+    pointer.onPointerMove(function(p) {
+        if (plot.entityNearest(p)) {
+        title.text(plot.entityNearest(p).datum.x + ", " + plot.entityNearest(p).datum.y);
+        }
+    });
+    pointer.attachTo(plot);
+
+    new Plottable.Interactions.PanZoom(xScale, null).attachTo(plot);
+
+    return new Plottable.Components.Table([
+        [yAxis, plot],
+        [null, xAxis],
+        [null, title],
+    ]);
+}
+
+function createPie(data) {
```

Comment:
yah - the logic and calculations are very different than the problem here, so im gonna punt for another PR